### ISSUE TITLE
Left navigation bar: Flyout improvements

### ISSUE THREAD

**Issue by jessover9000** [grafana/grafana#20978]
**Steps to reproduce**:
Try to open the flyout menu of the main navigation with a keyboard or if you have motor issues holding your mouse still.

**Actual Result**:
With the keyboard, you cannot get the navigation to fly out. That means that you don't get labels = don't know what the menu points mean or do and cannot access sub menus.
The flyout only happens on hover, meaning that it's hard to use it with motor impairments. 

**Expected Result**
Making the flyout appear on click might be more accessible and more easy to use. That would also solve the keyboard issues because the flyout could appear on pressing enter.

This was also criticised in a German-language accessibility study by a large German telco provider.

**Comment by ckbedwell** [comment]
This issue has been resolved.
